cirrus/server/Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN apt-get update -qqy \
 
 WORKDIR /application-services
 
-RUN git clone https://github.com/mozilla/application-services.git .
+RUN git clone https://github.com/jeddai/application-services.git .
+RUN git fetch && git checkout add-enrollment-status-metric-for-cirrus
```

Comment:
This is here to have the build run with the change as it has been written in application-services here: https://github.com/mozilla/application-services/pull/5891

Before merging it will be removed